packages/nuxt/src/core/nitro.ts

```diff
@@ -273,6 +273,12 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
   // Init nitro
   const nitro = await createNitro(nitroConfig)
 
+  // Set prerender-only options
+  nitro.options._config.storage ||= {}
+  nitro.options._config.storage['internal:nitro:prerender'] = { driver: 'memory' }
+  nitro.options._config.storage['internal:nitro:prerender:island'] = { driver: 'lruCache' }
+  nitro.options._config.storage['internal:nitro:prerender:payload'] = { driver: 'lruCache' }
```

Comment:
Since we are mounting and consuming them in nuxt maybe use `internal:nuxt`? Also is default lru params enough? (an explicit option could be nice).